env/posix/ocf_env_headers.h

```diff
@@ -3,6 +3,12 @@
  * SPDX-License-Identifier: BSD-3-Clause-Clear
  */
 
+/**
+ * @file
+ * @brief OCF libs and macros
+ *
+ * Header file with OCF common libraries and macros.
```

Comment:
Actually looks like this file is mostly to aggregate external header includes. 

Also, where is this file included? I don't see a single reference to it. And it looks like it would make sense to include it from env/posix/ocf_env.h.

Reply:
It's included in ocf_env.h.